.github/DISCUSSION_TEMPLATE/new-type.yml

```
@@ -1,59 +1,49 @@
-title: "Add <suggested new type name>"
+title: "<suggested new type name>"
 labels: ["New type"]
 body:
-- type: input
-  id: name
+- type: markdown
   attributes:
-    label: Type name
-    description: The name of the terms type to add, in Title Case.
-    placeholder: Business Privacy Policy
-  validations:
-    required: true
+    value: | 
+      Please share the details below to help the community to better understand your terms type suggestion.
 - type: textarea
   id: alternative-type-names
   attributes:
-    label: Alternative type names
+    label: What are the alternative names for this type?
     description: Known synonyms for the same terms type.
     placeholder: | 
       Business Users Privacy Policy
       Commercial Privacy Policy
-- type: input
-  id: writer
-  attributes:
-    label: Writer
-    description: The author of the document, in most cases the service provider itself.
-    placeholder: service provider
 - type: input
   id: audience
   attributes:
-    label: Audience
+    label: Who is the target audience for these terms?
     description: The targeted audience whose rights and duties are defined in the associated terms.
     placeholder: business user
 - type: input
   id: object
   attributes:
-    label: Object
-    description: The object of the commitment, i.e. the information or interaction whose handling will be constrained by the associated terms.
+    label: What is the object of commitment in these terms?
+    description: i.e. the information or interaction whose handling will be constrained by the associated terms.
     placeholder: personal data of business users
 - type: textarea
   id: examples
   attributes:
-    label: Examples
+    label: Can you provide examples of documents containing these terms?
```

The answer to this question would be “yes” or “no”, and we want examples instead 🙂 
```suggestion
    label: What are some documents containing these terms?
```